client/app/scripts/superdesk-authoring/styles/themes.less

```diff
@@ -51,6 +50,279 @@ body, html {
 	.box-sizing(border-box);
 	max-width: 900px;
 
+        header {
```

Comment:
:+1:  I was looking where to put this styles for header, and I found that in themes file there are a lot of definitions for editor! But good suggestion to put it in authoring.less, it makes more sens!